Test/Statistics/StatisticsDeviceTest.cs

```diff
@@ -61,6 +61,7 @@ public void Test()
 
             var count = 7;
 
+            Thread.Sleep(100);
```

Comment:
This change was put in its own commit since it had no relation with actual PR content, but was needed for CI to pass